feature-toggles/feature-toggles-impl/src/main/java/com/duckduckgo/feature/toggles/impl/metrics/RetentionMetricsAtbLifecyclePlugin.kt

```diff
@@ -85,8 +85,7 @@ class RetentionMetricsAtbLifecyclePlugin @Inject constructor(
 
     private fun daysBetweenTodayAnd(date: String): Long {
```

Comment:
There's one more copy of this function in [MetricsPixelInterceptor](https://github.com/duckduckgo/Android/blob/e8267daaf97981563608a355eb2d5b4c60ab5262/feature-toggles/feature-toggles-impl/src/main/java/com/duckduckgo/feature/toggles/impl/MetricPixelInterceptor.kt#L105) that we should also adjust, shouldn't we?

Reply:
This one looks fine to stay because it takes truncated days as parameter :+1: It would be great to add a small comment to the function parameter so that someone stumbling on this in the future will immediately understand the difference between the way we parse and use the various `enrollmentDate` instances.